cli/unstable_progress_bar.ts

```diff
@@ -0,0 +1,189 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+/**
+ * The options that are provided to a {@link createProgressBar} or
+ * {@link ProgressBarStream}.
+ */
+export interface ProgressBarOptions {
+  /**
+   * The offset size of the input if progress is resuming part way through.
+   * @default {0}
+   */
+  currentSize?: number;
```

Comment:
Maybe let's call it `value` like in [progress  HTMLElement](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress)?